src/_csrc/bcrypt.c

```diff
@@ -88,14 +87,13 @@ bcrypt_hashpass(const char *key, const char *salt, char *encrypted,
 	/* Check for minor versions */
 	switch ((minor = salt[1])) {
 	case 'a':
-		key_len = (u_int8_t)(strlen(key) + 1);
+		key_len = (u_int8_t)(key_len + 1);
 		break;
 	case 'b':
 		/* strlen() returns a size_t, but the function calls
```

Comment:
This comment needs to be updated

Reply:
Sorry, I haven't understood which comment are you referring to but if there is a comment to be updated I can do it gladly.

Reply:
The one on line 93 that this comment refers to. There's no longer a strlen anywhere in this function so it's not a relevant comment.